bundles/org.openhab.binding.philipsair/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,185 @@
+	</channel-type>
+	<channel-type id="iaql">
+		<item-type>Number</item-type>
+		<label>IAQL</label>
+		<description>Allergen index</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="aqit">
+		<item-type>Number</item-type>
+		<label>AQIT</label>
+		<description>AQI</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="err">
+		<item-type>String</item-type>
+		<label>Error Message</label>
+
+		<state readOnly="true" pattern="%s">
+			<options>
+				<option value="0">No error</option>
+				<option value="49408">No water</option>
+				<option value="32768">Water tank open</option>
+				<option value="49155">Clean pre-filter</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fltsts0">
+		<item-type>Number</item-type>
+		<label>Pre-filter</label>
+		<description>Pre-filter estimated clean up</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="wicksts">
+		<item-type>Number</item-type>
+		<label>Wick</label>
+		<description>Wick filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="fltsts2">
+		<item-type>Number</item-type>
+		<label>Carbon</label>
+		<description>Active carbon filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="fltsts1">
+		<item-type>Number</item-type>
+		<label>HEPA Filter</label>
+		<description>HEPA filter estimated lifetime</description>
+		<state readOnly="true" pattern="%d" />
+	</channel-type>
+	<channel-type id="rh">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>Current humidity</description>
+		<category>Humidity</category>
+		<state readOnly="true" pattern="%.1f %unit%" min="0" max="100" />
+	</channel-type>
+	<channel-type id="rhset">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity Setpoint</label>
+		<description>Humidity setpoint</description>
+		<category>Humidity</category>
+		<state readOnly="false" pattern="%.0f %unit%" min="40" max="70" step="10" />
+	</channel-type>
+	<channel-type id="temp">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current temperature</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<channel-type id="func">
+		<item-type>String</item-type>
+		<label>Function</label>
+		<description>P: Purification, PH: Purification and Humidification</description>
+		<state readOnly="false" pattern="%s">
+			<options>
+				<option value="P">Purification</option>
+				<option value="PH">Purification and Humidity</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="wl">
+		<item-type>Number</item-type>
+		<label>Water Level</label>
+		<description>Water tank level</description>
+		<state readOnly="true" pattern="%d" />

```

Comment:
Does this number have a range?

Reply:
Probably yes, I have to check with someone owns such device model.

Reply:
0-100

Reply:
> 0-100 (%)

Thanks! I will add it and will push all review changes pretty soon.